docs/src/pages/[platform]/components/rating/RatingPropControls.tsx

```diff
@@ -32,20 +32,38 @@ export const RatingPropControls: RatingPropControlsInterface = ({
   setFillColor,
   setEmptyColor,
 }) => {
+  function handleValueChange(value: string) {
+    if (value === '') {
+      setValue(0);
+    } else if (parseFloat(value)) {
+      setValue(parseFloat(value));
+    }
```

Comment:
Do we need an `else` here? Same question for ln 48

Reply:
I think for these demo controls it's fine without; we don't want to update the value if you start typing letters.